CMakeLists.txt

```diff
@@ -28,7 +28,7 @@
 # The SPIR-V headers from the SPIR-V Registry
 # https://www.khronos.org/registry/spir-v/
 #
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Vulkan-Headers is also a widely used KhronosGroup library so I'd say it's fine raise the min to 3.14:
https://github.com/KhronosGroup/Vulkan-Headers/blob/9f93cbe76abe9f6cb4a36df10b08fa3b78ae0027/CMakeLists.txt#L17